pkg/cidata/cidata.TEMPLATE.d/boot/27-discover-distribution.sh

```diff
@@ -0,0 +1,318 @@
+#!/bin/sh
+#    Salt Bootstrap - Generic Salt Bootstrap Script
+
+#    Copyright 2012-2021 SaltStack (saltproject.io)
+
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
```

Comment:
Please add a comment to clarify the origin (and the purpose) of this file

Reply:
And I'm feeling this script is too much complicated.. Just grepping /etc/os-release (lsb-release) might be enough